### PR TITLE
feat(CLI): Support context

### PR DESCRIPTION
**Pull request by gaocegege**
Ref #92 #510 

```
NAME:
   envd context - Manage envd contexts

USAGE:
   envd context command [command options] [arguments...]

COMMANDS:
   create   Create envd context
   ls       List envd contexts
   use      Use the specified envd context
   help, h  Shows a list of commands or help for one command

OPTIONS:
   --help, -h  show help (default: false)
```

Signed-off-by: Ce Gao <cegao@tensorchord.ai>

### CODE REVIEW

**Comment by gaocegege** [comment]
@VoVAllen PTAL

File: pkg/types/envd.go [hunk 1]
```diff
@@ -46,6 +46,24 @@ type EnvdInfo struct {
...
+	Name          string      `json:"name,omitempty"`
+	Builder       BuilderType `json:"builder,omitempty"`
+	BuilderSocket string      `json:"builder_socket,omitempty"`
+}
+
+type BuilderType string
+
+const (
+	BuilderTypeDocker     BuilderType = "docker-container"
+	BuilderTypeKubernetes BuilderType = "kube-pod"
```
Comment (VoVAllen) [inline pkg/types/envd.go:64]: Is this supported in this PR?

**Review by VoVAllen — APPROVED** [review]
LGTM

File: pkg/types/envd.go (hunk 1 above)
Comment (gaocegege) [reply pkg/types/envd.go:64]: No, it will be supported in the next PR.

**Pull request merged**